patches/spec.fetched.json/09-set-additional-prop-to-false-in-href.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/spec/oas3.patched/openapi/public/components/schemas/Href.yaml b/spec/oas3.patched/openapi/public/components/schemas/Href.yaml
+index bbe7e22..7cd92bb 100644
+--- a/spec/oas3.patched/openapi/public/components/schemas/Href.yaml
++++ b/spec/oas3.patched/openapi/public/components/schemas/Href.yaml
+@@ -4,3 +4,4 @@ properties:
+ required:
+ - href
+ type: object
++additionalProperties: false
```

Comment:
This is the right change IMO, but we should bear in mind that fixing it will likely uncover additional schema issues.